GW2EIEvtcParser/ParserHelpers/SkillIDs.cs

```diff
@@ -1319,36 +1348,84 @@ public static class SkillIDs
         public const long TrueNatureDemon = 51714;
         public const long LampBond = 51726;
         public const long AquaticDetainmentEffect = 51755;
+        public const long WaveOfForce = 51759;
+        public const long BodyOfFlame = 51879;
+        public const long ShatteredEarth = 51923;
         public const long EnragedQadim = 51933;
         public const long Waterlogged = 51935;
+        public const long QadimCC = 51943;
+        public const long SlashWyvern = 51958;
+        public const long VaporJet = 51965;
+        public const long AquaticBarrage = 51977;
+        public const long PatriarchCCJumpInAir = 51984;
+        public const long CycloneBurst = 51999;
+        public const long KenutAquaticAura = 52005;
         public const long PowerOfTheLamp = 52035;
         public const long EnragedWywern = 52040;
+        public const long Summon = 52054;
         public const long AugmentedPower = 52074;
         public const long ResistanceLavaElemental = 52075;
+        public const long JunkAbsorption = 52086;
+        public const long JunkFall2 = 52120;
+        public const long AquaticVortex = 52130;
+        public const long JunkTorrent = 52150;
+        public const long RupturedGround = 52161;
+        public const long Pulverize = 52173;
         public const long BrandedAccumulation = 52182;
         public const long AquaticAuraKenut = 52211;
         public const long FracturedAllied = 52213;
+        public const long Claw = 52221;
+        public const long FireWave2 = 52224;
+        public const long ShatteringImpact = 52242;
         public const long CAInvul = 52255;
+        public const long QadimRiposte = 52265;
         public const long Determined52271 = 52271;
+        public const long Swap = 52281;
+        public const long BigHit = 52310;
         public const long ConjuredSlash = 52325;
+        public const long SeismicStomp = 52330;
+        public const long FieryDance4 = 52383;
         public const long UnbearableFlames = 52408;
         public const long ArmUp = 52430;
+        public const long SeaOfFlame = 52461;
         public const long Parry = 52464;
+        public const long ElementalBreath = 52520;
         public const long FlameArmor = 52568;
+        public const long Inferno = 52587;
         public const long FierySurge = 52588;
+        public const long FieryDance1 = 52614;
         public const long EnragedTwinLargos = 52626;
+        public const long Tremor = 52656;
         public const long GreatswordPower = 52667;
+        public const long TailSwipe = 52705;
+        public const long FireBreath = 52726;
+        public const long WingBuffet = 52734;
         public const long ConjuredShield = 52754;
+        public const long NikareAquaticAura = 52779;
+        public const long ConjuredProtectionSAK = 52780;
+        public const long TidalPool2 = 52812;
```

Comment:
This one should be name TidalPoolSkill and the other one renamed to TidalPoolEffect